docs/design/aws_under_the_hood.md

```diff
@@ -0,0 +1,120 @@
+## Peeking under the hood of kubernetes on AWS
+
+We encourage you to use kube-up (or CloudFormation) to create a cluster.  But
+it is useful to know what is being created: for curiosity, to understand any
+problems that may arise, or if you have to create things manually because the
+scripts are unsuitable for any reason.  We don't recommend manual configuration
+(please file an issue and let us know what's missing if there's something you
+need) but sometimes it is the only option.
+
+This document sets out to document how kubernetes on AWS maps to AWS objects.
+Familiarity with AWS is assumed.
+
+### Top-level
+
+Kubernetes consists of a single master node, and a collection of minion nodes.
+Other documents describe the general architecture of Kubernetes (all nodes run
+Docker; the kubelet agent runs on each node and launches containers; the
+kube-proxy relays traffic between the nodes etc).
+
+By default on AWS:
+
+* Instances run Ubuntu 15.04.  It includes a sufficiently modern kernel to give
+  a good experience with Docker, it doesn't require a reboot.
+* By default we run btrfs as the filesystem on the nodes.  It is relatively
```

Comment:
For now it looks as if we're using aufs by default:
https://github.com/GoogleCloudPlatform/kubernetes/blob/f48543aba54e7edd545389fd104695463f0a354b/cluster/aws/config-default.sh#L30

Perhaps you have a PR switching to btrfs I'm not seeing?